app/controllers/miq_policy_controller/miq_actions.rb

```diff
@@ -70,7 +70,7 @@ def action_delete
     assert_privileges("action_delete")
     actions = []
     # showing 1 action, delete it
-    if params[:id].nil? || MiqAction.find_by_id(params[:id]).nil?
+    if params[:id].nil? || MiqAction.find(params[:id]).nil?
```

Comment:
this is wrong. `find` will raise an exception, you need `exists?` here or `exist?` I never know